boards/arm/thingy91_nrf9160/Kconfig

```diff
@@ -4,6 +4,19 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
+if BOARD_THINGY91_NRF9160 || BOARD_THINGY91_NRF9160_NS
+
+config BOARD_INIT_PRIORITY
+	int "Initialization priority for board configuration"
+	default 80
+	help
+	  The Thingy:91 board contains an ADP5360 PMIC that needs to be configured to
+	  set up the power domains on the board correctly. This happens during sys_init,
+	  and the PMIC setup must happen before the sensors are initialized in order
+	  to power them up in time.
```

Comment:
```suggestion
	  Board config initialization priority. This value should be lower than CONFIG_SENSOR_INIT_PRIORITY 
	  so that the ADP5360 PMIC is configured to set up the power domains on the board correctly
	  before the sensors are initialized. This is required to power the sensors in time.
```